### PR TITLE
foundation: add grants page

### PR DESCRIPTION
**Pull request by notmandatory**

### CODE REVIEW

File: docs/foundation/grants.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+lastUpdated: false
+---
+
+# Grants Program
+
+One of the primary purposes of the BDK Foundation is to support the ongoing development and maintenance of the BDK suite of open source software. To this end the BDK Foundation funds open source developers working on BDK and related projects via full-time and project based grants.
+
+## Grant Requirements
+
+Below is a list of general requirements the BDK Foundation looks for in grant applicants. These requirements may be demonstrated via any open source project you have contributed to. In particular through your public work on PR reviews, PRs, issues, team discussions, and current or prior side projects. We also consider work you've done as part of bitcoin development training programs (eg. Chaincode Lab, Base58, etc.) and related professional work experience.
```
Comment (reez) [inline docs/foundation/grants.md]: nit: just an "s" needed here - `Chaincode Lab` should be `Chaincode Labs`

**Comment by reez** [comment]
I love this, if I think of anything to add I'll let you know, but you did a really good job explaining things yet keeping it concise and simple and approachable 👍

**Comment by ValuedMammal** [comment]
> Supports the overall bitcoin freedom tech mission.

Obviously I agree with this sentiment, but I wonder how easy it will be to evaluate whether a potential contribution fits the overall bitcoin freedom tech mission? There's a lot of bitcoin-adjacent projects that I'm sure would meet this criteria, but where does bdk draw the line in choosing which projects to affiliate with and which ones probably don't belong, given that its also likely to change over time. I'm not exactly suggesting a change of wording, but it's something to consider (noting a similar confusion around the wording in BIP-0002 and what's considered BIP-worthy).

**Comment by evanlinjin** [comment]
I like how @kcalvinalvin puts it. Does this tech/project increase the trustlessness of Bitcoin? I think it's very easy to judge. I don't think we should base it in "freedom tech", that definition is just way too broad.

Edit: I think stating "Bitcoin mission" is self explanatory? We want to decrease trust required to interact with system, but we also want to onboard more users.

**Review by thunderbiscuit — APPROVED** [review]
ACK fb2dd3db37aa0badf3174e60c956c93b196e69fc. 

My only small nit would be that I think Rust as a programming language should be capitalized.

**Comment by notmandatory** [comment]
Thanks all for the feed back. I've made suggested fixes and updated the line on bitcoin's mission to read:

 - Supports the overall bitcoin mission, in particular to improve individuals financial privacy and autonomy.

I like the idea of "trustlessness" but I think above is a more general way to say bitcoin users shouldn't need to give up their privacy or be dependent/beholden to third parties.

File: docs/foundation/grants.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Project grants
+
+Project grants are for contributors who are able to focus part-time for at least three months on a project to improve BDK or a supporting project.
+
+Additional traits that are a plus for project grants:
+
+- A project proposal that clearly supports the BDK mission.
+- The project doesn't have to be coding, it could be research, testing, docs, etc.
+- The project can be completed is 3 months (one quarter).
```
Comment (oleonardolima) [inline docs/foundation/grants.md]: I guess there is a minor typo here:
```suggestion
- The project can be completed in 3 months (one quarter).
```

**Pull request merged**